UploadServiceLogo/index.ts

```diff
@@ -0,0 +1,44 @@
+import { Context } from "@azure/functions";
+import * as cors from "cors";
+import * as express from "express";
+import { cosmosdbInstance } from "../utils/cosmosdb";
+
+import {
+  SERVICE_COLLECTION_NAME,
+  ServiceModel
+} from "io-functions-commons/dist/src/models/service";
+import { secureExpressApp } from "io-functions-commons/dist/src/utils/express";
+import { setAppContext } from "io-functions-commons/dist/src/utils/middlewares/context_middleware";
+
+import createAzureFunctionHandler from "io-functions-express/dist/src/createAzureFunctionsHandler";
+
+import { getApiClient } from "../utils/apiclient";
+import { GetUploadServiceLogo } from "./handler";
+
+// Setup Express
+const app = express();
+secureExpressApp(app);
+
+const client = getApiClient();
+
+const serviceModel = new ServiceModel(
+  cosmosdbInstance.container(SERVICE_COLLECTION_NAME)
+);
+
+// Set up CORS (free access to the API from browser clients)

```

Comment:
Not needed.

Reply:
Fixed in [9bbe257](https://github.com/pagopa/io-functions-services/pull/68/commits/9bbe257016950fbe4605124203d3f558fd5f8b72)